docs/guides/cloud.rst

```diff
@@ -0,0 +1,125 @@
+.. _ref_guide_cloud:
+
+=====
+Cloud
+=====
+
+:edb-alt-title: Using EdgeDB Cloud (Beta)
+
+.. warning::
+
+    EdgeDB Cloud is currently in beta. This guide will not work unless you have
+    access. To request access, `join the waitlist </p/cloud-waitlist>`_ or join
+    `our Discord <https://discord.gg/umUueND6ag>`_ and request access there.
+
+EdgeDB Cloud is the easiest way to host your EdgeDB instance. We offer two ways
+to interact with EdgeDB Cloud: via our CLI or through a graphical web
+interface.
+
+CLI Cloud Interface
+===================
+
+To use Cloud via the CLI, first log in using :ref:`ref_cli_edgedb_cloud_login`.
+Once your login is successful, you will be able to create an instance using
+either :ref:`ref_cli_edgedb_instance_create` or
+:ref:`ref_cli_edgedb_project_init`, depending on whether you also want to
+create a local project linked to your instance.
+
+In either case, you will pass an instance name matching the EdgeDB Cloud
+instance name format: ``<github-username>/<instance-name>``
+
+To use ``edgedb instance create``:
+
+.. code-block:: bash
+
+    $ edgedb instance create <github-username>/<instance-name>
+
+To use ``edgedb project init``:
+
+.. code-block:: bash
+
+    $ edgedb project init \
+      --server-instance <github-username>/<instance-name>
+
+Alternatively, you can run ``edgedb project init`` *without* the
+``--server-instance`` option and enter an instance name in the
+``<github-username>/<instance-name>`` format when prompted interactively.
+
+
+Web Cloud Interface
+===================
+
+If you'd prefer, you can also manage your account via `the EdgeDB Cloud
+web-based GUI <https://cloud.edgedb.com/>`_.
+
+The first time you access the web UI, you will be prompted to log in.
```

Comment:
Devon, we're going to be rapidly iterating the cloud UI. Let's drop the screenshots entirely.